lib/tasks/customers.rake

```diff
@@ -15,4 +15,20 @@ namespace :customers do
       customer.update!(currency: currencies.first)
     end
   end
+
+  desc 'Set sync_with_provider field for existing customers'
+  task populate_sync_with_provider: :environment do
+    Organization.all.each do |org|
+      next if org&.stripe_payment_provider&.create_customers&.blank?
```

Comment:
Rails/SafeNavigationWithBlank: Avoid calling `blank?` with the safe navigation operator in conditionals.